apps/roam/src/components/DiscourseContextOverlay.tsx

```diff
@@ -138,29 +152,88 @@ const DiscourseContextOverlay = ({
     getInfo();
   }, [refresh, getInfo]);
 
-  // Suggestive Mode
-  const validTypes = useMemo(() => {
-    if (!discourseNode) return [];
-    const selfType = discourseNode.type;
-    const validRelations = relations.filter((relation) =>
-      [relation.source, relation.destination].includes(selfType),
+  const suggestionContextData = useMemo(() => {
+    const selfNode = discourseNode;
+
+    if (!selfNode || !selfNode.text || !selfNode.format) {
```

Comment:
same as [this](https://github.com/DiscourseGraphs/discourse-graph/pull/163#discussion_r2096426436)